FPGA/CPU_GARAGE.qsf

```diff
@@ -242,7 +242,26 @@ set_global_assignment -name IP_SEARCH_PATHS ../source/common
 set_global_assignment -name ENABLE_SIGNALTAP ON
 set_global_assignment -name USE_SIGNALTAP_FILE output_files/stp1.stp
 set_global_assignment -name SEARCH_PATH ../source/common
-
```

Comment:
@amichaibd adds UART files for FPGA target.

Reply:
looks good